luigi/scheduler.py

```diff
@@ -224,6 +224,9 @@ def prune(self, config):
         if self.last_active + config.worker_disconnect_delay < time.time():
             return True
 
+    def assistant(self):
```

Comment:
I still haven't learned about the best practive for when to use properties in python or not. Hmm ...